Update the docs on passthrough proxy (#20072)
keycloak/keycloak: docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc

    <artifactId>keycloak-admin-client-jee</artifactId>
    <version>22.0.0</version>
</dependency>
----

= Passthrough proxy mode changes

> Keycloak's proxy configuration setting for mode *passthrough* does now longer parse HTTP forwarding headers in the request, as when a proxy forwards an HTTPS connection in passthrough mode, a proxy is unable to add, remove or update HTTP headers.

Reviewer: @ahus1 Should be like this, right?

```suggestion
Keycloak's proxy configuration setting for mode *passthrough* no longer parses HTTP forwarding headers in the request, as when a proxy forwards an HTTPS connection in passthrough mode, a proxy is unable to add, remove or update HTTP headers.
```

Reviewer: Or just `no longer parses`?

Author: Fixed in latest commit, ready for review.
